src/DOMBuilder.js

```diff
@@ -35,6 +35,12 @@ module.exports = (function(){
     popElement: function() {
       this.currentNode = this.currentNode.parentNode;
     },
+    popElements: function(n) {
+      while (n>0) {
```

Comment:
since we're terminating on n=0, we can say `while(n--)` here, so that we don't need the decrement in the body.